admin/src/components/Pagination.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+export default function Pagination({ pageCount, setCurrentPage, currentPage, className }) {
+  const pages = Array(pageCount)
+    .fill(0)
+    .map((_, i) => i + 1);
+  const previousPages = pages.slice(currentPage - Math.min(4, currentPage), currentPage - 1);
+  const nextPages = pages.slice(currentPage, currentPage + Math.min(3, pages.length - currentPage));
+
+  return (
```

Comment:
Il manque un return to the beginning et un go to the end pour la pagination
+ essaie de faire ressembler le style a notre pagi de base

Reply:
Ca je vais le refaire.